registry/verify-namespace-owner/verify_namespace_owner_test.go

```diff
@@ -72,13 +72,21 @@ func TestVerifyNamespaceOwner(t *testing.T) {
 
 			it("fails if add-if-missing is false", func() {
 				tk.On("GetInput", "add-if-missing").Return("", false)
+				tk.On("GetInputList", "blocked_namespaces").Return([]string{""}, false)
+				Expect(owner.VerifyNamespaceOwner(tk, o, r, s)).
+					To(MatchError("::error ::invalid namespace test-namespace"))
+			})
 
+			it("fails if namespace is blocked", func() {
+				tk.On("GetInput", "add-if-missing").Return("", false)
+				tk.On("GetInputList", "blocked_namespaces").Return([]string{""}, false)
```

Comment:
I think this needs to return a list that includes the namespace used in the test. so:

 `.Return([]string{"test-owner"},`

Reply:
thanks a lot, @jkutner , have tried to fix it in the next commit, thanks